frontend/packages/patternfly/.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Just curious, does this follow `patternfly` standards that are different from our main `.eslintrc`?

Reply:
@benjaminapetersen, this is the main .eslintrc. I found that if I didn't explicitly add it here, then it didn't follow the same eslint rules as the rest of the console.

Reply:
Interesting. I'm guessing our `package.json` `yarn lint` command must be aimed at a subset of the `/frontend` directory.  Prob something we want to track down.